content/en/guide/v10/forms.md

```diff
@@ -5,37 +5,46 @@ description: 'How to build awesome forms in Preact that work anywhere.'
 
 # Forms
 
-Forms in Preact work much the same as they do in HTML. You render a control, and attach an event listener to it.
+Forms in Preact are largely the same as HTML forms.
 
-The main difference is that in most cases the `value` is not controlled by the DOM node, but by Preact.
+Much like in HTML forms, forms in Preact both work by rendering input elements and listening for events triggered as the form is modified. The primary difference when rendering forms with Preact is that the value of each form input is generally controlled by Preact.
 
 ---
 
 <div><toc></toc></div>
 
 ---
 
-## Controlled & Uncontrolled Components
+## Controlled Component and Uncontrolled Component
 
-When talking about form controls you'll often encounter the words "Controlled Component" and "Uncontrolled Component". The description refers to the way data flow is handled. The DOM has a bidirectional data flow, because every form control will manage the user input themselves. A simple text input will always update it's value when a user typed into it.
+You'll often encounter the terms "Controlled Component" and "Uncontrolled Component" in documentation about working with forms in Preact. These terms refer to the way data flow is handled. The DOM has bidirectional data flow where each form control manages its own state and user input. For example, when typing into a text input element, the text is updated without requiring additional logic to store and display values.
```

Comment:
Considering consistency with following, I think `"Controlled" Component and "Uncontrolled" Component` is better than `"Controlled Component" and "Uncontrolled Component"`.  I think "Controlled" and  "Uncontrolled" refer to the way data flow is handled.


> A "Controlled" Component is a component where state must be manually managed by listening for events and passing new values as props. An "Uncontrolled" Component is a component that manages its own internal state, handling events and updating itself automatically.